__tests__/app.test.js

```diff
@@ -34,7 +34,31 @@ describe("GET /api/topics", () => {
       .expect(404)
       .then((response) => {
         const error = response.body;
-        expect(error.msg).toBe("path not found")
-    });
+        expect(error.msg).toBe("path not found");
+      });
+  });
+});
+
+describe("GET /api/articles", () => {
+  test("200: responds with an array of all articles in order of date descending", () => {
+    return request(app)
+    .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.allArticles).toHaveLength(37);
```

Comment:
This is a lot of articles. Are you using the test data for your tests? 👀

Reply:
Thanks, I've altered it to the test file, I noticed this on another function that I was working on whilst I was waiting for the pull request to be checked and I've altered it on that one when I noticed too.